corehq/ex-submodules/casexml/apps/case/tests/test_bugs.py

```diff
@@ -133,12 +133,11 @@ def testSubmitToDeletedCase(self):
         deleted_doc_type = 'CommCareCase-Deleted'
         [xform, [case]] = FormProcessorInterface.post_case_blocks([
             CaseBlock(create=True, case_id=case_id, user_id='whatever',
-                      update={'foo': 'bar'}).as_xml()
```

Comment:
Why is this necessary - I don't think we should ever change the doc_type via a form submission.